blocking-elements.js

```diff
@@ -312,7 +355,7 @@
      * @returns {boolean}
      * @private
      */
-    static[_isInert](element) {
+    [_isInert](element) {
       return element.inert;
```

Comment:
Lol no, this remained from the times I was undecided if to rely or not on the attribute...will remove it in favor of direct checks 👌